### PR TITLE
Create initial version of LastPass guidance

### PR DESCRIPTION
**Pull request by cybersquirrel**
To help get some guidance out to support our deployment (as per #25 )

### CODE REVIEW

**Comment by SteveMarshall** [comment]
Hey both, this looks good in general, but at the moment, I'm just rolling LastPass out to Ops teams and some of your gang. As part of the move from Rattic to LastPass, I'm hoping to get a BA involved to understand who else might need access. So: we probably won't know for a few weeks how wide we want the user base to be, or what the process would be for people requesting access. 

Given that, do you want to leave this here and review when that happens? Or what?

**Comment by joelgsamuel** [comment]
@SteveMarshall I want to use this as a learning/training exercise for an apprentice in cyber security and have her lead this. I think we can skip things like "who should use this" and "how to get access" and focus more on the uses (or non-uses) so I think we can do this now and then add later?

 cc/ @cybersquirrel

**Comment by SteveMarshall** [comment]
@joelgsamuel That sounds fine to me, yeah 👍

**Comment by KayleighAcourt** [comment]
Mature draft for everyone to review :-)

This is just for the users of LastPass. More to come about people in MOJ who will maintain/run it (@SteveMarshall 's team) and 'defend' (Defensive Security Operations)

File: guides/using-lastpass.md [hunk 1]
```diff
@@ -4,50 +4,101 @@ expires: 2019-09-01
...
+* SMS-based (a one-time code sent via SMS)
 
-You will be sent an email inviting you to create your LastPass account. You need to create a master password - this is the only password you'll need to remember, but it is important that it's long and strong. It also has to be unique.
+If you don’t have an MOJ-issued work smartphone you may use a personal device for MFA.
 
-You next need to install the LastPass browser extension. ( How to do this? Self Service if you aren't on dev build?) 
+### Sharing passwords
 
-(something about only using work email address)
+To share a password [create a 'shared folder' in the LastPass Vault](https://support.logmeininc.com/lastpass/help/manage-lastpass-teams-shared-folders-users-lp010061)
```
Comment (cybersquirrel) [inline guides/using-lastpass.md]: Something here about ensuring you only share with people who genuinely need to see the shared password / regularly reviewing this?

File: guides/using-lastpass.md [hunk 2]
```diff
@@ -4,50 +4,101 @@ expires: 2019-09-01
...
 
-## The product
+## What is LastPass?
 
-LastPass is an online password management tool that we make available to you to help you store and share passwords. Using it means you no longer need to remember dozens of passwords, just a single master password. It keeps all your website logins protected, helps with creating new passwords, password sharing where required, and ensuring passwords are suitably strong.
+LastPass is an online password management tool that we make available to you to help you create, store and share passwords. Using it means you no longer need to remember dozens of passwords, just a single master password. It keeps all your website logins protected, helps with creating new ‘strong’ passwords and passwording sharing when required.
 
-The LastPass software will be installed on your browsers as a browser extension. As you go to work-related sites, save the credentials to your LastPass vault. LastPass will then autofill those passwords the next time you go to log in.
+LastPass is available as a browser extension for popular browsers and as well as a full software suite (for use outside of browsers) for Microsoft Windows and Apple macOS.
```
Comment (cybersquirrel) [inline guides/using-lastpass.md]: Something about 'contact service desk to get the extension installed'

Comment (joelgsamuel) [reply guides/using-lastpass.md]: I think we need @SteveMarshall 's view on 'support'

Browser extension:
I don't _think_ there are any restrictions set by G-Suite/Google policy by DSD that stop people self-installing the browser extension. No idea about on IE/Firefox on DOM1.

'Fat' Windows app:
That needs Atos to wrap it into SelfService?

'Fat' macOS app (https://apps.apple.com/us/app/lastpass/id926036361?ls=1&mt=12):
I again don't know MDM controls for MOJDT macOS devices. Can people self install this? If so, which Apple ID do they use for the 'purchase' (etc)

Comment (joelgsamuel) [reply guides/using-lastpass.md]: bump @SteveMarshall

Comment (SteveMarshall) [reply guides/using-lastpass.md]: @joelgsamuel I'm honestly not sure what you need from me here? Personally, I don't care which route… And I'm not sure I have any way to decide which would work, either?

Comment (joelgsamuel) [reply guides/using-lastpass.md]: @SteveMarshall if we're suggesting fat clients, people need to be able to install that (admin rights in local OS likely required) which needs some collaboration with DSD. I guess we're looking for steer on what you are organising with them, or not.

(I think we prefer fat clients)

Comment (SteveMarshall) [reply guides/using-lastpass.md]: I'm organising nothing; I'm not being responsible for rolling it out formally across the organisation, just making licenses available for people who need infrastructure.

**Comment by joelgsamuel** [comment]
@KayleighAcourt you can now `Squash and merge`

**Pull request merged**